gear-programs/vft-gateway/src/services/mod.rs

```diff
@@ -330,7 +339,7 @@ where
                 gear_bridge_builtin: config.gear_bridge_builtin,
                 receiver_contract_address: config.receiver_contract_address,
                 admin: exec_context.actor_id(),
-                eth_client: config.eth_client,
+                eth_client: exec_context.actor_id(),
```

Comment:
I think we can get it from config as maybe this contract will be deployed after `ethereum-event-client` and address will already be known

Reply:
ok